src/sdk/auth.js

```diff
@@ -34,6 +35,7 @@ export class OCAuthCore
         this.logoutEndPoint = logoutEndPoint;
         this.redirectUri = redirectUri;
         this.referralCode = referralCode;
+        this.isSandBox = isSandBox;
```

Comment:
ahhhh actually I think it shouldn't be here ... it should be the backend of auth service. 
like it should be configured in the sandbox instance of the auth service to redirect user to that page instead. Sorry we don't need to change that in the SDK.